backend/src/main/java/me/performancereservation/api/ManagerPerformanceController.java

```diff
@@ -0,0 +1,144 @@
+package me.performancereservation.api;
+
+
+import lombok.RequiredArgsConstructor;
+import me.performancereservation.domain.file.FileService;
+import me.performancereservation.domain.file.dto.UploadFileResponse;
+import me.performancereservation.domain.performance.dto.performance.request.PerformanceCreateRequest;
+import me.performancereservation.domain.performance.dto.performance.request.PerformanceUpdateRequest;
+import me.performancereservation.domain.performance.dto.performance.response.PerformanceManagerDetailResponse;
+import me.performancereservation.domain.performance.dto.performance.response.PerformanceManagerListResponse;
+import me.performancereservation.domain.performance.service.PerformanceScheduleService;
+import me.performancereservation.domain.performance.service.PerformanceService;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+@RestController
+@RequestMapping("/api/v1/managers")
+@RequiredArgsConstructor
+public class ManagerPerformanceController {
+    private final PerformanceService performanceService;
+    private final PerformanceScheduleService performanceScheduleService;
+    private final FileService fileService;
+
+    /** 공연자 자신의 공연 목록 페이지 호출
+     *
+     * 시큐리티 개발후 인증 시스템에서 매니저 Id 추출하는 방향으로 수정 필요
+     * @param pageable
+     * @param managerId
+     * @return 200 + Page<PerformanceManagerListResponse>
+     */
+    @GetMapping("/performances")
+    public ResponseEntity<Page<PerformanceManagerListResponse>> getPerformances(
+            @PageableDefault(
+                    size=10,
+                    sort = "performanceDate",
+                    direction = Sort.Direction.DESC) Pageable pageable,
+            Long managerId) {
+        Page<PerformanceManagerListResponse> performanceManagerResponses = performanceService.getPerformanceManagerList(pageable, managerId);
```

Comment:
```suggestion
        Page<PerformanceManagerListResponse> result = performanceService.getPerformanceManagerList(pageable, managerId);
```

사소한거긴한데 `performanceManagerResponses`는 `Page<PerformanceManagerListResponse>`자체가 1개의 응답을 나타내는 객체라서 복수형보다는 단수형인 response나 result가 더 적절해보여요

Reply:
처음에 performanceManagerListResponse 로 썼다가 이름이 너무 긴 것 같아서 어떻게 써야 데이터가 여러 개 있는 List의미를 전달할 수 있을까 하고 Responses 로 바꿨었습니다. 

근데 듣고 보니 응답 자체는 한 개라 단수형으로 쓰는 게 더 적절해 보이네요!